pkg/models/import.go

```diff
@@ -359,6 +359,9 @@ func ImportDocumentData(
 				if err2 := inTx(ctx, tx, 0, true); err2 != nil {
 					return 0, errors.Join(ErrAlreadyInDatabase, err2)
 				}
```

Comment:
Does this statement need to be removed? I still get the same errors.

Reply:
No, the error canceled the whole transaction so it was not possible to write anything into the transaction afterwards. This is now circumvented by isolating the storing of the document with a save point.
This results in only rolling back to this and keep the transaction intact for further writes.

This should work now.